how-to/reports/comments-and-tags.md

```diff
@@ -11,29 +11,39 @@ This section describes how to add comments and tags to Reports and individual Sn
+Also, a new column appears on the left side of the Farm Explorer Report/Snapshots grid with the speech bubble icon displayed for the rows which have comments.
 
 To add a comment to the selected Snapshot or Farm Explorer Report row, select the __Add__ button in the top menu's __Comments__ tab.
-You can also add a comment by selecting the beforementioned first column field.
+You can also add a comment by clicking the beforementioned first column field.
 Notice the speech bubble icon and the tooltip appearing when you hover the mouse over the first column of the Farm Explorer Report grid or Snapshots grid.
 
-The top menu's __Comments__ tab enables you to easily __Add__, __Edit__, and __Delete__ comments. The same options are available in the contextual menu that appears when you right-click the first column of the Farm Explorer Report or Snapshots grid. 
+The top menu's __Comments__ tab enables you to easily __Add__, __Edit__, and __Delete__ comments. The same options are available in the contextual menu that appears when you right-click a row.
 
-To view all comments you have created, navigate to the top menu's __Comments__ tab an select the __All__ option. Comments overview grid opens with the columns __Author__, __Comment__, and __Last Modified__ visible and grouped by __Path__. When you double-click a comment from the grid, associated Farm Explorer Report or Snapshot to which the comment belongs is displayed.  
+To view all comments you have created, navigate to the top menu's __Comments__ tab and select the __Go to Report__ option. The __All Comments__ report opens with the columns __Author__, __Comment__, and __Last Modified__ displayed and grouped by __Path__. When you double-click a comment from the grid, associated Farm Explorer Report or Snapshot to which the comment belongs is displayed. The same action can also be done by clicking the __Go to Comment__ button in the top menu __Comments__ tab.  
+
+The __All Comments__ report can easily be exported by selecting the __Export as PDF/XLSX/DOCX__ option from the __Home__ tab in the top menu.
+
+To include your comments in the generated documentation navigate to the __Backstage__ and on the __Configuration__ screen and select __Options__. Under __Report Options__ check the __Include comments in documentation__ option. Your comments will now be visible in the generated documentation. 
 
-To include your comments in the generated documentation navigate to the __Backstage Configuration Screen__ and select __Options__. Under __Report Options__ check the __Include comments in Documentation__ option. Your comments will now be visible in the generated documentation. 
 
 ### __Tags__
 
 In the Permissions Explorer, you can find another useful feature - adding tags to Securable Objects.
 
 Tags can be added Live but also in all created Snapshots.
```

Comment:
unclear what "in all created snapshots" mean. 
maybe: "...but also when exploring a previously created snapshot?"